features/step_definitions/web_steps.rb

```diff
@@ -57,6 +57,15 @@ def with_scope(locator)
   expect(page).send should, have
 end
 
+Then /^(?:I )should see the select "([^"]*)"(?: with options "([^"]+)")?$/ do |label, with_options|
```

Comment:
Sorry to bother you again, but making "I " and "with options ..." optional is also unnecessary. Let's remove those to keep the steps simpler and more readable.

Reply:
done